go.mod

```diff
@@ -123,7 +123,7 @@ require (
 
 replace (
 	github.com/crossplane/crossplane-runtime => github.com/verrazzano/crossplane-runtime v0.17.0-1
-	github.com/crossplane/oam-kubernetes-runtime => github.com/verrazzano/oam-kubernetes-runtime v0.3.3-1
+	github.com/crossplane/oam-kubernetes-runtime => github.com/verrazzano/oam-kubernetes-runtime vz.8147
```

Comment:
Is this a temporary change ?

Reply:
Yes, Once merged there will be tag in similar format like previously and then checked into VZ